tests/js/spec/views/organizationDetails/organizationsDetails.spec.jsx

```diff
@@ -147,6 +64,89 @@ describe('OrganizationDetails', function () {
     wrapper.update();
     expect(getTeamsMock).toHaveBeenCalled();
     expect(getProjectsMock).toHaveBeenCalled();
-    expect(wrapper.find('OrganizationContextContainer').prop('detailed')).toBe(false);
+  });
+
+  describe('pending deletion', () => {
+    it('should render a restoration prompt', async function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/',
+        body: TestStubs.Organization({
+          slug: 'org-slug',
+          status: {
+            id: 'pending_deletion',
+            name: 'pending deletion',
+          },
+        }),
+      });
+      wrapper = mountWithTheme(
+        <OrganizationDetails params={{orgId: 'org-slug'}} location={{}} routes={[]} />,
+        TestStubs.routerContext()
+      );
+      await tick();
+      await tick();
+      wrapper.update();
+      expect(wrapper.text()).toContain('Deletion Scheduled');
+      expect(wrapper.text()).toContain(
```

Comment:
this reminds me a lot of RTL 👏😀

Reply:
While it may look like I rewrote these tests, I actually just moved things around 😆